scripts/add_uniprot_sequences.py

```
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+"""
+Script to add UniProt protein sequences to mzIdentML fixture files.
+This updates DBSequence elements that are missing sequence data.
+"""
+
+import re
```

Flake8 (7.3.0)

[error] 're' imported but unused

(F401)